src/EnliteMonolog/Service/MonologServiceFactory.php

```diff
@@ -57,6 +58,10 @@ public function createLogger($container, MonologOptions $options)
             $logger->pushProcessor($this->createProcessor($container, $processor));
         }
 
+        if (class_exists('\Monolog\ErrorHandler') && $options->isErrorHandler()) {
```

Comment:
yeah i agree. never look back! it would be a breaking change, though. do you know the maintainer of this repo? we could submit another merge request with that breaking change to see what they think.